[calibration] Store extra information in `info.txt` file

giordano opened the pull request:
Address first point of #1244

giordano commented on .github/workflows/calibration.yaml:
```diff
@@ -163,6 +163,13 @@ jobs:
+          # Print info.txt file with some useful information
+          info_txt="${task_output_dir}/info.txt"
+          git -C working-directory log --pretty=format:"%H%x09%ad%x09%s" --date=iso-strict -n 1 > "${info_txt}"
+          echo "${GITHUB_SERVER_URL}/${GITHUB_REPOSITORY}/actions/runs/${GITHUB_RUN_ID}" >> "${info_txt}"
```
Note: logs are automatically deleted after ~3 months or so, so this line can be useful for reference for some time, but not forever.

tamuri replied:
I think that's okay - it's for an easy way to get to see stdout when something goes wrong.

tamuri commented on .github/workflows/calibration.yaml:
```diff
@@ -163,6 +163,13 @@ jobs:
           python3 "${{ needs.setup.outputs.worktree_path }}/src/scripts/calibration_analyses/analysis_scripts/process.py" "${task_output_dir}" "${RUN_NAME}" "${{ needs.setup.outputs.worktree_path }}/resources"
+
+          # Print info.txt file with some useful information
+          info_txt="${task_output_dir}/info.txt"
```
I think this would put the file in the `022_long_run_all_diseases_process` directory. Can we put it in the parent (I guess `${{ needs.setup.outputs.output_dir }`)?

giordano replied:
You're right :facepalm: 
```suggestion
          info_txt="${{ needs.setup.outputs.output_dir }}/info.txt"
```

giordano commented:
I was wondering, if the idea was to use this file in case something goes wrong, maybe we should create it early on, in the setup step, instead of at the end of postprocessing, where it's too late if anything failed before?

tamuri commented:
Good point - you're right. With the info.txt file we could show something on the generated webpage, enough to indicate "we tried to run analyses on this commit, and it failed".

giordano commented:
Ok, I moved the writing of the file to its own step in the setup job, and addressed all other silly mistakes I made yesterday.  Hopefully now it should be all good :smiling_face_with_tear:
